README.adoc

```diff
@@ -11,12 +11,13 @@ projects: []
 :toc:
 :icons: font
 :source-highlighter: prettify
-:project_id: draft-gs-template
+:project_id: gs-spring-cloud-gcp
 This guide walks you through the process of creating a Spring application.
```

Comment:
I think it's an artifact of staging and not controllable here, but @gregturn should know for certain.